lib/rules/no-else-return.js

```diff
@@ -208,15 +205,9 @@ module.exports = {
                 return;
             }
 
-            for (consequents = []; node.type === "IfStatement"; node = node.alternate) {
```

Comment:
no idea why this was added, looks like it's only there to break `else if` reporting?